### ISSUE TITLE
Enumerations are broken

### ISSUE THREAD

**Issue by jhartman** [scalagwt/scalagwt-scala#42]
Not sure if this is intended or not. I'm using milestone 2, and scalac-client is equivalent to scalac in that ant file. I have my build split up into server and client classpaths so that I can separate things out.

[scalac-client] Compiling 5 source files to /Users/jhartman/code/orgchart/war/WEB-INF/jribble
[scalac-client] /Users/jhartman/code/orgchart/shared/src/josh/shared/Seniority.scala:3: error: not found: type Enumeration
[scalac-client] object Seniority extends Enumeration {
[scalac-client]                          ^
[scalac-client] /Users/jhartman/code/orgchart/shared/src/josh/shared/Seniority.scala:4: error: not found: type Value
[scalac-client]   type Seniority = Value

**Comment by gkossakowski** [comment]
Yeah, Enumeration is not supported in Scala+GWT (for client code) due to its implementation depending on reflection.

Until different implementation of scala.Enumeration is provided one needs to fallback to Java's enumeration.
